rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/integration/SearchTest.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
```

Comment:
AFAIK, you are not employee of Siemens and it is not 2017. Please update the copyright accordingly :-)